model/Entity/TasksLogsStats.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2017 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ *
+ */
+
+namespace oat\taoTaskQueue\model\Entity;
+
+use JsonSerializable;
+
+class TasksLogsStats implements JsonSerializable
```

Comment:
I feel this class a bit overkill. It doesn't contain heavy logic, the data can be returned without this class as well.

Reply:
If you keep it, make it similar like the TaskLogEntity. (When you will get the raw data from the broker, you can map the result to this entity)

Reply:
I will keep it and mapp from raw data as the TaskLogEntity.